lib/generators/templates/rpush.rb

```diff
@@ -1,5 +1,7 @@
 Rpush.configure do |config|
 
+  # Set multiple database configuration with active_record
+  # config.database = {}
```

Comment:
Please put this below `config.client`, similarly to `redis_options`. Also, I think we should make it clearer where the docs for possible values are / what the value of this is used for.

Reply:
Unfortunately the `database` option must be set before the client is initialized. I added some comments about the possible values to use.

Reply:
I don't think setting the client type is the point at which the client is initialized, is it? You're just building a config object here.

Reply:
Nope, you're right, we do init the client at that point. 🤦‍♂️